content/stub.xhtml

```diff
@@ -479,9 +476,9 @@
             window.frameElement.setAttribute("tooltip", "aHTMLTooltip");
           let mainWindow = topMail3Pane(window);
           let willExpand = parseInt(params.willExpand);
-          let msgHdrs = [msgUriToMsgHdr(x) for (x of params.urls.split(","))];
-          msgHdrs = msgHdrs.filter(function (x) x != null);
-          msgHdrs = msgHdrs.filter(function (x) x.messageId);
+          let msgHdrs = params.urls.split(",").map(x => msgUriToMsgHdr(x));
+          msgHdrs = msgHdrs.filter(x => x != null);
+          msgHdrs = msgHdrs.filter(x => x.messageId);
```

Comment:
nit: you could clean this up and do; `params.url.split(...).filter(x => x != null && x.messageId)`